cmd/cstorvolumeclaim/start.go

```diff
@@ -27,23 +31,27 @@ import (
 	clientset "github.com/openebs/maya/pkg/client/generated/clientset/versioned"
 	informers "github.com/openebs/maya/pkg/client/generated/informers/externalversions"
 	ndmclientset "github.com/openebs/maya/pkg/client/generated/openebs.io/ndm/v1alpha1/clientset/internalclientset"
-	"github.com/openebs/maya/pkg/signals"
+	leader "github.com/openebs/maya/pkg/kubernetes/leaderelection"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 var (
-	masterURL  string
-	kubeconfig string
+	masterURL              string
+	kubeconfig             string
+	leaderElectionLockName = "cvc-controller-leader"
```

Comment:
`leaderElectionLockName` -- > It is actually the name of `Lease` resource that is used for selecting a leader among the contending pods, right ? The pod who updates the `Lease` resource first is the leader.
What about following ? 
```
// leaseResourceName is the name of the `Lease` resource.
// The contending pods try to put their details in the Lease resource and the only one
// pod who is able to do that becomes the leader. 
leaseResourceName = "cvc-controller-lease"
```